aaf_logging/launch/topics.launch

```diff
@@ -0,0 +1,72 @@
+<launch>
+
+<arg name="machine" default="localhost"/>
+<arg name="user" default="" />
+
+<arg name="logged_topics" value="
+    /rosout_filtered
+    /tf
+    /robot_pose
+    /cmd_vel
+    /goal
+    /mileage
+    /motor_status
+    /head/actual_state
+    /head/cmd_light_state
+    /head/commanded_state
+    /barrier_status
+    /battery_state
+    /bumper
+    /charger_status
+    /rfid
+    /diagnostics
+    /SetPTUState/goal
+    /ResetPtu/goal
+    /EBC/parameter_updates
+    /Charger/parameter_updates
+    /topological_navigation/Route
+    /topological_navigation/Statistics
+    /current_node
+    /current_edge
+    /closest_node
+    /do_backtrack/goal
+    /speak/goal
+    /mary_tts/speak
+    /strands_emails/goal
+    /strands_image_tweets/goal
+    /pygame_player_negotiation
+    /chargingServer/goal
+    /chargingServer/result
+    /chargingServer/cancel
+    /docking/goal
+    /docking/result
+    /docking/cancel
+    /undocking/goal
+    /undocking/result
+    /undocking/cancel
+    /map_updates
+    /move_base/NavfnROS/plan
+    /move_base/current_goal
+    /move_base/DWAPlannerROS/global_plan
+    /move_base/DWAPlannerROS/local_plan
+    /move_base/goal
+    /wait_node/goal
+    /wait_node/result
+    /wait_node/cancel
+"/>
+
+<!-- Excluded topics (due to already being logged)
+    /monitored_navigation/monitored_nav_event
+    /monitored_navigation/srv_pause_requested
+    /monitored_navigation/stuck_on_carpet
+    /monitored_navigation/pause_requested
+    /execute_policy_mode/Statistics
+    /task_executor/events
+    /current_schedule
+-->
+
+<machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>
+
+<node pkg="mongodb_log" type="mongodb_log.py" name="aaf_logger" output="screen" respawn="true" args="--nodename-prefix=aaf_logger $(arg logged_topics)"/>
+
+</launch>
```

Comment:
ATM, we should only run the bash script. This is here for when it's possible to launch it using a launch file, which feels more ergonomic.